private_repo/ansible/inventory/dev/Core/common.yml

```diff
@@ -51,7 +51,7 @@ certs_badge_image: ""                # Image of the certs badge e.g value is "su
 certs_badge_root_org_id: ""          # Root org ID for which the certs will be created e.g value is "0124229474659643111"
 certs_badge_issuer_name: ""          # The issuer of the certs name e.g value is "Government of karnataka"
 certs_badge_issuer_url:  ""          # Issuer organisation official URL e.g value is "https://testdomain.in"
-cert_service_cloud_storage_type: azure  
+cert_service_cloud_storage_type: azure
```

Comment:
If we know other clouds are not supported, can we move this to all.yml?

Reply:
don't know about that. Have to check all the options. I think this worth another PR ?

Reply:
Not sure.. maybe we should check dev - Satish Nirmalkar